### ISSUE TITLE
1.5.x versions break phpcs with Drupal coding standards

### ISSUE THREAD

**Issue by ccjjmartin** [AtomLinter/linter-phpcs#214]
Using 1.5.x versions and Drupal coding standards the linter breaks.  With an error message about an unexpected error has occurred.  See next comment for screenshot of a similar error.

My solution was to back up to 1.4.1 for now which works with the exact same local environment setup.  

UPDATE: Tested this again 11 days later and it appears to be working now ...  Very strange maybe something in the node world was broken when I did my install a few days ago.

As the next comment mentions below you can upgrade using `apm install linter-phpcs@1.5.3` or whatever the latest version is.

I had cloned this repo down, so after pointing to the 1.5.3 commit, I wiped the node_modules directory in my ~/.atom/packages/linter-phpcs directory using `rm -Rf node_modules` and did a fresh `npm install` which appears to have fixed things.

If this breaks again I will reopen the issue hopefully with steps to reproduce.

**Comment by entozoon** [comment]
Same situation for me, with CakePHP standards:

![erroe](http://i.imgur.com/c5N3vGV.png)

As mentioned, rolling back works around the issue for now, via:
```
apm install linter-phpcs@1.4.1
```

**Comment by briancwelch** [comment]
👍 Same for me but using WordPress standards.

**Comment by ccjjmartin** [comment]
@entozoon @briancwelch I was attempting to put further research into what was causing the bug today but version 1.5.3 appears to be working again on my machine.  Hopefully this is the case for both you too.

**Comment by briancwelch** [comment]
Thank you @ccjjmartin, can confirm that everything is fine now.

**Comment by jparcher** [comment]
I'm running version 1.5.3 and I'm still getting the same issues as @entozoon. Reverting to 1.4.1 fixed the issue for me.

**Comment by entozoon** [comment]
I'm still getting it @ccjjmartin, same error as in previous screenshot with 1.5.3 whenever there is a linting mistake.

**Comment by ccjjmartin** [comment]
@entozoon Now that I look at your error a little closer it says "Invalid point encountered".  This is actually a separate issue than I was encountering and seems like a real bug as opposed to what I encountered (which was probably an upgrade issue that could be solved by reinstalling the module).  It might be worth opening a separate issue to get it some attention.

**Comment by ccjjmartin** [comment]
For anyone who happens to come across this later.  I ran into some problems again but believe it is related to the minimap linter package not necessarily this one (the bug showed it was unable to spawn a child process).

I disabled the minimap-linter and the bug went away.  Enabled the minimap again and it was still gone ...
